src/tools/arakoon_logger.ml

```diff
@@ -67,17 +67,23 @@ let console_logger =
 
 
 let format_message
-      ~hostname ~pid ~component
-      ~ts ~seqnum
+      ~hostname ~pid ~thread_id ~component
+      ~(ts:Core.Time.t) ~seqnum
       ~section ~level ~lines
   =
+  let thread_id = 0 in
   Printf.sprintf
-    "%s - %s - %i/0 - %s - %i - %s - %s"
+    "%s - %s - %i/%i - %s - %i - %s - %s"
     (let open Unix in
-     let tm = Core.Time_internal.to_tm ts in
+
+     let (fts:float) = Core.Time.to_span_since_epoch ts
+                       |> Core.Time.Span.to_sec
+     in
+     let tm = Unix.localtime fts in
      let offset =
-       Core.Time.utc_offset ~zone:Core.Time.Zone.local ts
-       |> Core.Span.to_parts
+       let zone = Lazy.force Core.Time.Zone.local in
+       Core.Time.utc_offset ~zone ts
+       |> Core.Time.Span.to_parts
      in
      Printf.sprintf
        "%04d-%02d-%02d %02d:%02d:%02d %d %c%02d%02d"
```

Comment:
I know it's unrelated, but can you also change the `%d` for printing the microseconds to `%06d`?
Just remembered something about @jeroenmaelbrancke's log parsing having some problems with this, so he would like to see that changed...